ansible_collections/arista/avd/plugins/modules/yaml_templates_to_facts.py

```diff
@@ -77,12 +81,45 @@
       Autodetects data format based on file suffix. '.yml', '.yaml' -> YAML, default -> JSON
     required: false
     type: str
+  mode:
+    description: File mode for dest file.
+    required: false
+    type: str
   template_output:
     description: |
       If true the output data will be run through another jinja2 rendering before returning.
       This is to resolve any input values with inline jinja using variables/facts set by the input templates.
     required: false
     type: bool
+  conversion_mode:
+    description:
+      - Run data conversion in either "warning", "info", "debug", "quiet" or "disabled" mode.
+      - Conversion will perform type conversion of input variables as defined in the schema.
+      - Conversion is intended to help the user to identify minor issues with the input data, while still allowing the data to be validated.
+      - During conversion, messages will generated with information about the host(s) and key(s) which required conversion.
+      - conversion_mode:disabled means that conversion will not run.
+      - conversion_mode:warning will produce warning messages.
+      - conversion_mode:info will produce regular log messages.
+      - conversion_mode:debug will produce hidden messages viewable with -v.
+      - conversion_mode:quiet will not produce any messages.
+    required: false
+    default: "debug"
```

Comment:
so by default we would run debug mode?

Reply:
by default we produce the messages that the user can see if they use -v

Reply:
should we have some kind of message telling the user to use `-v` to see the messages if anything goes "wrong" like ansible / pytest do to  help you to get more info?

Reply:
added